index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <script src='https://kit.fontawesome.com/a076d05399.js' crossorigin='anonymous'></script>
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <header class="justify-between">
+      <div id="logo-placeholder">
+        <img src="./images/logo.png" alt="logo">
```

Comment:
- [ ] Hello @AsumVictor, great effort working on your Porfolio project, but currently according to the template you are using we only have the hamburger icon on the right hand side and there is no logo on the left hand side. I would kindly advise you stick to the Figma design template as much as possible as this will show the confidence in your ability to replicate any given design.
- [ ] please kindly remove the logo image on the left hand side